### ISSUE TITLE
FMI3.0 support

### ISSUE THREAD

**Issue by keiligch** [NTNU-IHB/FMI4j#140]
Hi, is it planned to update FMI4j to support the new FMI3.0 standard, which will be released soon?

**Comment by chrbertsch** [comment]
Hi @markaren: any news on FMI 3 Support?

**Comment by markaren** [comment]
No, I don't work with the JVM any longer so I think it's time to archive this repository.
